addons/website/static/src/js/editor/snippets.editor.js

```diff
@@ -187,4 +187,20 @@ weSnippetEditor.Class.include({
         });
     },
 });
+
+weSnippetEditor.Editor.include({
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+    /**
+     * @override
+     */
+    _isEmptyAndRemovable($el, editor) {
+        editor = editor || $el.data('snippet-editor');
+        return this._super(...arguments) || ($el.children('.s_parallax_bg').length === $el.children().length
```

Comment:
I think it should be the same behavior with shape, video, etc. Not only the parallax background.

Reply:
Good catch!